flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "skip-icons-check": true,
+    "skip-appstream-check": true
```

Comment:
You're shipping a metainfo,any reason for skipping the validation?

Reply:
no good reason, reverted. But I know it doesn't like the lack of icons.